deploy/helm/charts/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+name: openebs-cstor
+description: CStor-Operator helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 2.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 2.1.0
+
+dependencies:
+  - name: openebs-ndm
+    version: "0.8.0"
```

Comment:
nit: 1.0.1

Reply:
```yaml
dependencies:
  - name: openebs-ndm
    version: "1.0.1"
    repository: "https://openebs.github.io/node-disk-manager"
    condition: openebsNDM.enabled
```
and in values.yaml just add 

```yaml
# If false, openebs NDM sub-chart will not be installed
openebsNDM:
  enabled: true

```
We can use the `condition` field in case if want to skip the installation of dependency charts https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies